src/frontend/state/GlobalState.tsx

```diff
@@ -790,9 +790,13 @@ class GlobalState extends PureComponent<Props> {
       libraryStatus,
       sidebarCollapsed,
       hideChangelogsOnStartup,
-      lastChangelogShown
+      lastChangelogShown,
+      language
     } = this.state
 
+    const isRTL = RTL_LANGUAGES.includes(language)
+    document.body.classList.toggle('isRTL', isRTL)
```

Comment:
we need the `direction: rtl` in the body to fix one issue with the scrollbar overlapping the content

there seems to be a bug in chrome that setting `direction: rtl` in the body doesn't move the scrollbar to the left side of the screen, but it seems to be a bug, we can't do much

it would work if the scroll were in a nested element, but then that breaks controllers